app/models/pagination.rb

```diff
@@ -0,0 +1,15 @@
+class Pagination < Dry::Struct
```

Comment:
Can we use `Grape::Entity` instead of Dry for this? There is a basic example I used for the health check in `app/api/datastore/entities/healthcheck.rb`

With entities you can do transformations as well, etc. So probably we only need a `PaginatedCollection` entity (or if you prefer 2 entities) that exposes the structure and attributes we want.

Reply:
Ah good call. I'd not realised these. Will use that instead.

Reply:
I've made this change. I find the way Entities handle collections irritating, and am not convinced the presentation layer adds anything useful at the moment.